includes/layout/functions.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Useful functions
+ *
+ * @since 1.0.0
+ * @package RBEA Templates
+ */
+
+if ( ! function_exists( 'rbea_block_templates_log' ) ) :
+
+	/**
+	 * Log
+	 *
+	 * @param string $message Log message.
+	 */
+	function rbea_block_templates_log( $message = '' ) {
+		if ( rbea_block_templates_doing_wp_cli() ) {
+			WP_CLI::line( $message );
```

Comment:
@vibhavsalelkar we haven't registered any wp cli command, please remove this code.

Reply:
removed unwanted codes, comments and logs